### PR TITLE
Added sunshine cocktail

### PR DESCRIPTION
**Pull request by ragrag**

### CODE REVIEW

File: src/recipes/sunshine-cocktail.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    "1/2 oz. French vermouth",
+    "1 oz. pineapple juice",
+    "1 dash grenadine "
+  ],
+  "directions": [
+    "Shake ingredients well with cracked ice.",
+    "Lightly strain into a chilled cocktail glass."
+  ],
+  "image": "sunshine-cocktail.jpg",
+  "keywords": ["sunshine", "cocktail", "beack"]
```
Comment (alfg) [inline src/recipes/sunshine-cocktail.json]: Should this be "beach"?

Comment (ragrag) [reply src/recipes/sunshine-cocktail.json]: oh, yes it should be "beach", sorry for the inconvenience.

Comment (alfg) [reply src/recipes/sunshine-cocktail.json]: No worries. I updated.

**Comment by alfg** [comment]
Thanks!

**Pull request merged**